src/Markdown.MAML/Renderer/MamlRenderer.cs

```diff
@@ -87,10 +91,12 @@ private static XElement CreateSyntaxItem(MamlSyntax syntaxItem, MamlCommand comm
         {
             return new XElement(commandNS + "syntaxItem", 
                     new XElement(mamlNS + "name", command.Name),
-                    syntaxItem.Parameters.Select(param => CreateParameter(param, true)));
+                    syntaxItem.Parameters.Select(param => CreateParameter(param, isSyntax: true)));
         }
 
-        private static XElement CreateParameter(MamlParameter param, bool syntax)
+        // We pass a syntax flag due to a weired quirk inside <syntax>:
+        // we don't add [switch] info to make it appear good
```

Comment:
1) The comment is about the implementation detail of the help engine that explains why there is such a mess around this code:

```
!isSyntax || !isSwitchParameter ? new XElement(commandNS + "parameterValue",	...
```

It should be near the code

2) it's a good idea to make a doc comment that explains why we need `isSyntax` parameter at the first place here. Please use the doc format with

```
/// <summary>
```

and such. In Visual Studio it's `///` and it will auto-generate the stub.